package-lock.json

```diff
@@ -1,8 +1,8988 @@
 {
   "name": "tdr-auth-server",
   "version": "0.1.0",
-  "lockfileVersion": 1,
```

Comment:
Any idea why this file has changed so much when package.json hasn't changed?

Reply:
No idea, but my local npm install has decided it should update it, for no apparent reason

Reply:
I think it'll be safe to just discard these changes. If the package.json hasn't changed it should be fine.